src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -410,18 +410,18 @@
               (assoc-in [:wallet :transactions] transaction-details)
               (assoc-in [:wallet :ui :send :transaction-ids] transaction-ids))
       :fx [[:dispatch
-            [:wallet/wizard-navigate-forward
-             {:current-screen :screen/wallet.transaction-confirmation
-              :flow-id        :wallet-send-flow}]]]})))
+            [:wallet/end-transaction-flow]]]})))
```

Comment:
Shouldn't we update the `flow_config` of bridge and send?

Reply:
yes good point, I'll double check that. I see you had it in the similar pr so will check with that too 🙏